energistream/stream.py

```diff
@@ -558,6 +558,11 @@ def _request(self, rel_resource_path, params,
                    'client.').format(response_code)
             raise(TypeError(msg))
 
+
+    def sensors_unique(self):
```

Comment:
In the short future we need to drop the groupId from the sensors_unique(), in the long run we need a separate query to just return the sensors.  Tony had concerns that we were pulling time_zone off of the group, but it appears as though this is one of the 'drop_cols' and is remapped on the sensor query.